Rakefile

```diff
@@ -98,3 +102,31 @@ task :release do
     end
   end
 end
+
+namespace :local do
+  desc "show help"
+  task :version do
+    require_relative 'mrblib/mruby-cli/version'
+    puts "mruby-cli #{MRubyCLI::Version::VERSION}"
+  end
+end
+
+def is_in_a_docker_container?
+  `grep -q docker /proc/self/cgroup`
+  $?.success?
+end
+
+Rake.application.tasks.each do |task|
+  next if ENV["RUN_RAKE_TASK_LOCALLY"]
```

Comment:
I'm +1 having a prefix `MRUBY_CLI`. I think I prefer `MRUBY_CLI_LOCAL` since the local tasks may or may not be a "build" per se or `MRUBY_CLI_LOCAL_TASK`?